src/frontEnd/reactChecks/constants.js

```diff
@@ -1,3 +1,7 @@
 const { createObject } = require('../../utils');
 
-module.exports = createObject(['REDUX_RECOMPOSE']);
+module.exports.reactMetrics = createObject(['REDUX_RECOMPOSE', 'INDEX_LINES', 'LAYOUT_LINES', 'I18N']);
```

Comment:
many projects are not using `REDUX_RECOMPOSE` instead we are using Context with globalReducer, and reducers for screens, what about this for the future?

Reply:
Yup you are 100% right. We should design a way ('if necessary') to measure context and global reducers. Above that maybe we can have the two metrics and apply each one to a specific project. What do you think?

Reply:
yeap, agree 100%.
But those metrics can be optional, just one could be necessary. for example: if the projects uses redux_recompose its ok, if uses context its ok, but if uses none: bad metric. What do you think?